packages/flutter/lib/src/material/switch.dart

```diff
@@ -385,6 +387,40 @@ class Switch extends StatelessWidget {
   /// the [Switch] track has no outline by default.
   final MaterialStateProperty<Color?>? trackOutlineColor;
 
+  /// {@template flutter.material.switch.trackOutlineWidth}
+  /// The outline width of this [Switch]'s track.
+  ///
+  /// Resolved in the following states:
+  ///  * [MaterialState.selected].
+  ///  * [MaterialState.hovered].
+  ///  * [MaterialState.focused].
+  ///  * [MaterialState.disabled].
+  ///
+  /// {@tool snippet}
+  /// This example resolves the [trackOutlineWidth] based on the current
+  /// [MaterialState] of the [Switch], providing a different outline width when it is
+  /// [MaterialState.disabled].
+  ///
+  /// ```dart
+  /// Switch(
+  ///   value: true,
+  ///   onChanged: (_) => true,
```

Comment:
onChanged is `void Function(bool value)`
```suggestion
  ///   onChanged: (bool value) { },
```